activestorage/test/controllers/representations/proxy_controller_test.rb

```diff
@@ -126,15 +128,19 @@ class ActiveStorage::Representations::ProxyControllerWithPreviewsWithStrictLoadi
   end
 
   test "showing existing preview record" do
+    transformations = { resize_to_limit: [100, 100] }
+
     with_strict_loading_by_default do
       get rails_blob_representation_proxy_url(
         filename: @blob.filename,
         signed_blob_id: @blob.signed_id,
-        variation_key: ActiveStorage::Variation.encode(resize_to_limit: [100, 100]))
+        variation_key: ActiveStorage::Variation.encode(transformations))
     end
 
     assert_response :ok
     assert_match(/^inline/, response.headers["Content-Disposition"])
+    assert_equal @blob.representation(transformations).download, response.body
+
     @blob.reload # became free of strict_loading?
     assert_predicate @blob.preview_image, :attached?
 
```

Comment:
I'm not gonna lie, I don't understand the purpose of reading the image by doing `image = read_image(@blob.preview_image.variant(resize_to_limit: [100, 100]).processed)`

This has already been tested. I believe what's important is to check that the "rack response" have : 
1- the proper status code
2- proper http headers
3- proper body

If possible I would like to remove them now that I've added: `assert_equal @blob.representation(transformations).download, response.body`